version/oraclejdk10.sh

```diff
@@ -5,3 +5,4 @@ echo oracle-java10-installer shared/accepted-oracle-license-v1-1 select true | d
 sudo add-apt-repository -y ppa:linuxuprising/java
 sudo apt-get update
 sudo apt-get install -y oracle-java10-installer
+sudo apt-get install -y oracle-java10-set-default
```

Comment:
where is this documented?

Reply:
i just saw it here https://stackoverflow.com/questions/49507160/how-to-install-jdk-10-under-ubuntu

Reply:
Let's just use update-alternatives here

Reply:
ok